entrypoint.sh

```diff
@@ -21,9 +21,85 @@ if [ "$AVD" ]; then
     socat tcp-listen:5555,bind=127.0.0.1,fork tcp:$AVD & SOCAT_PID=$!
 fi
 
-start
+swarmUrl="$JENKINS_MASTER_URL/swarm/swarm-client.jar"
+swarmPath="/swarm-client.jar"
 
-if [ "$AVD_PID" ]; then
-    kill $SOCAT_PID
-fi
+startTime=$(date +%s)
+period=10
+success=1
+
+attemptTimeout() {
+  echo "One more attempt in $period seconds"
+  sleep $period
+}
+
+while [ $(( startTime + SWARM_RESPONSE_TIMEOUT )) -gt "$(date +%s)" ]; do
```

Comment:
why do we execute test and download for every start? Idea was to keep existing jar if any and only in case of connect failure redownload..